.github/actions/setup-linux/action.yml

```diff
@@ -52,14 +52,17 @@ runs:
       working-directory: ${{ inputs.repository }}
       run: |
         RUNNER_ARTIFACT_DIR="${RUNNER_TEMP}/artifacts"
+        sudo rm -rf "${RUNNER_ARTIFACT_DIR}"
         mkdir -p "${RUNNER_ARTIFACT_DIR}"
```

Comment:
Hmm, shouldn't we do something like `RUNNER_ARTIFACT_DIR=$(mktemp -p ${RUNNER_TEMP} artifacts-XXXXX)`